script/dist-info.js

```diff
@@ -36,6 +36,14 @@ function getOSXZipPath() {
   return path.join(getDistPath(), '..', getOSXZipName())
 }
 
+function getLinuxZipName() {
+  return `${productName}.zip`
```

Comment:
The currently-supported formats from `yarn run package` are RPM, DEB and AppImage. I'm not sure where this is coming from.